OpenRA.Mods.Common/Traits/SmokeTrailWhenDamaged.cs

```diff
@@ -37,19 +39,14 @@ class SmokeTrailWhenDamaged : ITick
 	{
 		readonly SmokeTrailWhenDamagedInfo info;
 		readonly BodyOrientation body;
-		readonly int getFacing;
```

Comment:
I am assuming that this was a bug, because it doesn't make sense to call this `getFacing` if its not returning the current facing.